include/vt/scope-guard/detail/platform.h

```diff
@@ -10,9 +10,27 @@
 
 
 #if defined(__cpp_lib_uncaught_exceptions) && __cpp_lib_uncaught_exceptions >= 201411
+#if defined(__APPLE__)
+
+#include <Availability.h>
+
+#if defined(__MAC_OS_X_VERSION_MIN_REQUIRED) && __MAC_OS_X_VERSION_MIN_REQUIRED >= 101200 || \
```

Comment:
nit: `101200` -> `__MAC_10_12` and so on

Reply:
apple suggest not to use it without `defined` checks so this is a bad suggestion